conf/provisioning/dashboards/nuodb/nuodb-ops-overview.json

```diff
@@ -1000,159 +1045,26 @@
       "targets": [
         {
           "alias": "$col",
-          "dsType": "influxdb",
           "groupBy": [
             {
               "params": [
-                "10s"
+                "$__interval"
               ],
               "type": "time"
             },
             {
               "params": [
-                "none"
+                "null"
               ],
               "type": "fill"
             }
           ],
-          "hide": false,
-          "measurement": "Commits",
           "orderByTime": "ASC",
           "policy": "default",
-          "query": "SELECT sum(\"rate\") AS \"TPS\", count(\"rate\") AS \"# TEs\" FROM \"Commits\" WHERE (\"nodetype\" = 'Transaction')  AND \"db\" =~ /^$database$/ AND \"host\" =~ /^$host$/ AND $timeFilter GROUP BY time($__interval) fill(none)",
+          "query": "SELECT sum(\"rate\") AS \"TPS\", count(\"rate\") AS \"# TEs\" FROM (SELECT mean(\"rate\") AS rate FROM \"Commits\" WHERE (\"nodetype\" = 'Transaction' AND \"db\" =~ /^$database$/ AND \"host\" =~ /^$host$/) AND $timeFilter GROUP BY time($__interval), \"host\" , \"db\", \"$pid\" fill(null)) WHERE $timeFilter GROUP BY time($__interval)",
```

Comment:
I assume that this is the relevant change and everything else is automatically changed by Grafana once the dashboard is exported?

Reply:
yes,  there were a few other's removed that were disabled.   The issue with dashboards is that the version is constantly changing.   An older grafana can load a newer dashboard,  however, some fields might be missing, additional fields will be discarded on load.